features/local-font-access.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-28
-name: Local Font Access
-description: TODO
+name: Local font access
```

I think this might be more to the point:

```suggestion
name: Local fonts
```

(similarly we'd change the file name to `local-fonts.yml`)